packages/instantsearch.js/src/middlewares/__tests__/createInsightsMiddleware.ts

```diff
@@ -872,7 +872,7 @@ See documentation: https://www.algolia.com/doc/guides/building-search-ui/going-f
         eventType: 'click',
         payload: {
           hello: 'world',
-        },
+        } as any,
```

Comment:
Can you walk me through why this needs to be typed as any?

Reply:
because the payload is now correctly typed, and these aren't correct payloads